contributor-guides/for-staff.md

```diff
@@ -33,6 +33,7 @@ These are some tools some of us find helpful:
 * [Markdown Here](https://markdown-here.com/) - for formatting emails written using Markdown
 * [Github Learning lab](https://lab.github.com/) - for learning git or refresh your skill
 * [Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf) - for a list of the most common used git commands
+* [Rewriting Git History - Amend, Reword, Delete, Reorder, Squash and Split](https://www.youtube.com/watch?v=ElRzTuYln0M) - for making the git history as clear and understandable as possible
```

Comment:
Would it be better to link to https://www.themoderncoder.com/rewriting-git-history/ which, in turn, links to that video?

Reply:
That's a very interesting question. Personally, when I started searching, I didn't look for a text based medium, I went straight for video.

Reply:
I think it would be good to the link from @ericherman, as that includes the video. This way those who prefer tutorials in written form can read the text, and those who like videos can click on that straight away.

Reply:
The youtube page includes the link to the text, so those those who prefer videos view directly, and those who like tutorials in written form can can click on that straight away. ;)

Reply:
> The youtube page includes the link to the text

Okay, then I'm fine either way. I'd still lean towards the article over youtube, but it matters less if they are both cross linked.